activiti-services/activiti-services-events/src/main/java/org/activiti/services/events/MessageProducerActivitiEventListener.java

```diff
@@ -24,10 +29,34 @@ public MessageProducerActivitiEventListener(ProcessEngineChannels producer,
 
     @Override
     public void onEvent(ActivitiEvent event) {
+        CommandContext currentCommandContext = Context.getCommandContext();
```

Comment:
I added the integration test(using MessageProducerActivitiEventListener, MessageProducerCommandContextCloseListener), instead of a unit test.
https://github.com/Activiti/activiti-cloud-starters/blob/daisuke-1407-audit/activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/AuditServiceIT.java

Should I also write unit tests?